.github/workflows/cicd-test.yml

```
@@ -0,0 +1,56 @@
+      - "v*"
+  pull_request:
+jobs:
+  test:
+    runs-on: ubuntu-22.04
+    env:
+      CGO_ENABLED: 0
+      GH_ACCESS_TOKEN: ${{ secrets.GH_ACCESS_TOKEN }}
+      GO_PRIVATE: github.com/threatmate/*
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)